owners/src/repo/virtual_repo.js

```diff
@@ -143,11 +143,12 @@ module.exports = class VirtualRepository extends Repository {
     return await this.cache.readFile(repoPath, async () => {
       let contents = '';
       try {
-        contents = await this.github.getFileContents(relativePath);
+        const result = await this.github.getFileContents(relativePath);
```

Comment:
Nit: destructure

Reply:
`contents` needs to exist outside of the try/catch scope. Is it possible to destructure into an already-defined variable?

Reply:
Technically yes, but it requires you to early declare the `sha` variable too:

```
let sha;
({ contents, sha} = await this.github.getFileContents(relativePath);
```

I find this ugly, so we can skip.